cluster/cluster-up-cpu

```diff
@@ -9,12 +9,17 @@ echo "Using Project:      ${PROJECT}"
 echo "Using Zone:         ${ZONE}"
 echo "Using Cluster Name: ${CLUSTER_NAME}"
 echo "Using K8S Version:  ${K8S_VERSION}"
+echo "Number of Nodes:    ${NUM_K8S_NODES}"
+
+export PARALLELISM="$((3 * ${NUM_K8S_NODES}))"
+
+check_gcloud_exists
 
 # Create a Kubernetes cluster
 # Note, we require Intel Broadwells since they are a bit newer, and can provide
 # up to a 30% speedup, since we're so CPU bound.
 gcloud beta container clusters create \
-  --num-nodes 3 \
+  --num-nodes $NUM_K8S_NODES \
   --machine-type n1-standard-8 \
```

Comment:
if these are standard-8's, parallelism should probably be higher, in 6-8

Reply:
Changed to 8

Reply:
(I'll do some test also)